src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/Validators/ValidationBehavior.shared.cs

```diff
@@ -11,6 +13,9 @@ namespace Xamarin.CommunityToolkit.Behaviors.Internals
 	/// </summary>
 	public abstract class ValidationBehavior : BaseBehavior<VisualElement>
 	{
+		const string validVisualState = "Valid";
+		const string invalidVisualState = "Invalid";
```

Comment:
Please make them public, people can refer to these to not have to rely on magic strings